cfpb/cfgov-refresh
# Featured menu content fix

Changes wrapper element for featured menu content module's text from `<p>` to `<div>` to prevent HTML validation issues when text contains paragraph tags. Addresses GHE issue 3302

## Changes

- Updates wrapper element in `featured-menu-content.html`

## Testing

1. Check out branch and navigate to http://localhost:8000/
2. Verify that HTML validation errors no longer appear for the  `.m-featured-menu-content` sections 

## Screenshots

### Before
<img width="1035" alt="9ac0db80-344c-11e9-91c6-3478cf552742" src="https://user-images.githubusercontent.com/778171/61720419-96f6f500-ad1b-11e9-9bcd-989da686acef.png">

### After
<img width="649" alt="Screen Shot 2019-07-23 at 7 26 33 AM" src="https://user-images.githubusercontent.com/778171/61720409-9199aa80-ad1b-11e9-9210-947b4e4a3d20.png">
